scripts/ci/dump_diagnostics.sh

```diff
@@ -7,6 +7,9 @@ namespace="$1"
 echo "Dumping CRD"
 kubectl get crd mongodbcommunity.mongodbcommunity.mongodb.com -o yaml > diagnostics/crd.yaml
 
+echo "Dumping Pod list"
+kubectl get pods > diagnostics/pod-list.yaml
```

Comment:
I believe you might have missed `-o yaml` switch in this command. Alternatively, I'd recommend changing the file extension to txt.

Reply:
Oh, yeah the purpose was indeed to use a "normal" `kubectl get pods` and not `-o yaml`. Let me change the output file to `.txt`.

Reply:
Done: https://github.com/mongodb/mongodb-kubernetes-operator/pull/1184/commits/e07dbd22e55999d2d06bec1d7b2c678e6c5b24f0!